src/Mentat.UI/Controllers/MentorController.cs

```diff
@@ -10,21 +13,41 @@ public IActionResult Index()
             return View();
         }
 
-        public IActionResult AddAssignment()
+
+        public IActionResult AddAssignment(Models.Assignment assignment)
         {
+ 
             return View();
         }
 
         [HttpPost]
         [ValidateAntiForgeryToken]
         public ActionResult SubmitForm(Models.Assignment assignment)
```

Comment:
Not a required change, but please try this ...
```csharp
public ActionResult SumitForm([FromForm] Assignment assignment)
{
    ... remove all the stuff before if (ModelState.IsValid)
} 
```

At a minimum please remove 'Models.' before the word assignment. This should be doable by importing the name-space. Same thing on line 17